### PR TITLE
High Freq Stop Word List while Querying & Indexing

### PR DESCRIPTION
**Pull request by PathakPratik**

### CODE REVIEW

File: src/main/java/QueryIndex.java [hunk 1]
```diff
@@ -74,14 +78,27 @@ public int compare(String o1, String o2) {
...
      */
     public void queryMap(HashMap<Integer,String> queries,
-                         String indexDirectoryLocation) throws IOException, ParseException {
+                         String indexDirectoryLocation) throws Exception {
         Directory directory = FSDirectory.open(Paths.get(indexDirectoryLocation));
         DirectoryReader directoryReader = DirectoryReader.open(directory);
         IndexReader indexReader = DirectoryReader.open(directory);
         IndexSearcher indexSearcher = new IndexSearcher(directoryReader);
         indexSearcher.setSimilarity(AnalyzerSimilarityFactory.getSimilarity(mSimilarityString));
+
```
Comment (jackgilbride999) [inline src/main/java/QueryIndex.java:89]: I don't fully understand the following code block, what's the distinction between the "500" passes in and the "70000" in the comparison? Do they represent different things?

Comment (PathakPratik) [reply src/main/java/QueryIndex.java:89]: Actually, I am getting top 500 high freq words and then checking their freq is greater than 70000. After that check, 482 words remain. 500 is just something that needs to be passed to getHighFreqTerms function

Comment (jackgilbride999) [reply src/main/java/QueryIndex.java:89]: Makes sense, thanks! That seems like it should work to me so, not sure what the issue is unfortunately.

**Comment by PathakPratik** [comment]
- Generating high-frequency words
- Using high-frequency words in stop word list during indexing as well as querying
- Cleaned up the code

**Review by OvrK12 — APPROVED** [review]
looks good to me

**Pull request merged**